lib/PuppeteerSharp/Target.cs

```diff
@@ -56,6 +56,25 @@ internal Target(TargetInfo targetInfo, Func<Task<CDPSession>> sessionFactory, Br
         /// <value>The target identifier.</value>
         public string TargetId => _targetInfo.TargetId;
 
+        /// <summary>
+        /// Get the target that opened this target
+        /// </summary>
+        /// <remarks>
+        /// Top-level targets return <c>null</c>.
+        /// </remarks>
+        public Target Opener
+        {
+            get
+            {
+                var opener = _targetInfo.OpenerId;
+                if (opener == null)
+                {
+                    return null;
+                }
+                return Browser.TargetsMap.GetValueOrDefault(opener);

```

Comment:
Can’t we solve this property in one line by sending a possible null to GetValueOrDefault?

Reply:
`GetValueOrDefault` uses `IDictionary<TKey, TValue>.TryGetValue` and will throw an `ArgumentNullException` exception if the argument is null